2wr-app/src/api/base-api-instance.js

```diff
@@ -1,16 +1,22 @@
-import axios from 'redaxios';
-import { getAuthInstance } from '../auth';
+import axios from "axios";
```

Comment:
If possible, can we stick to using `redaxios`? 

We are using `redaxios` because it uses browser's fetch api which means we can support offline using service worker / PWA features. Regular `axios` uses XMLHttpRequest which can't be intercepted using service worker. We don't have the service worker configured to do this yet but I think images would be a good place to use one of the workbox strategies that involves a cache https://developers.google.com/web/tools/workbox/modules/workbox-strategies . We can configure the service worker / workbox stuff in a separate PR.

Reply:
The problem is that redaxios doesn't work with the blob downloads (it's one of incompatibilities). We're going to need to think about offline for photos too in this case (other issue). but I'd be happy to opt into full axios just for the photo download if that's possible. Let's discuss it. 

Let me look at the redaxios repo and see this can be acconmplished with a PR there or perhaps just use fetch to do it.

Reply:
Ah, interesting. Yeah if redaxios doesn't work we could use fetch directly.

Reply:
redaxios works in every case but the photo upload (a bug in what they're expecting, never expect a file object, so it's json-izing it). I fixed this by removing axios, re-enabling redaxios, and impemented the photoApi without redaxios (used fetch natively so that you intercepting fetch will work).